modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -1156,6 +1156,89 @@ public void testLimitSearchToContentsCreatedWithinAPeriodOfTime()
 		_assertSearchIgnoreRelevance("[Coca Cola, Pepsi Cola, Sprite]");
 	}
 
+	@Test
+	public void testLimitSearchToHeadVersion() throws Exception {
+		_updateConfigurationJSON(
+			"queryConfiguration", JSONUtil.put("applyIndexerClauses", false));
+
+		_journalArticles.add(
+			JournalTestUtil.addArticle(
+				_group.getGroupId(), 0,
+				PortalUtil.getClassNameId(JournalArticle.class),
+				HashMapBuilder.put(
+					LocaleUtil.US, "Article 1.0"
+				).build(),
+				null,
+				HashMapBuilder.put(
+					LocaleUtil.US, ""
+				).build(),
+				LocaleUtil.getSiteDefault(), false, true, _serviceContext));
+
+		_journalArticles.set(
+			0,
+			JournalTestUtil.updateArticle(
+				_journalArticles.get(0), "Article 1.1"));
+
+		_journalArticles.set(
+			0,
+			JournalTestUtil.updateArticle(
+				_journalArticles.get(0), "Article 1.2"));
+
+		_updateElementInstancesJSON(
+			new Object[] {
+				HashMapBuilder.<String, Object>put(
+					"boost", 1
+				).put(
+					"fields", SXPBlueprintSearchResultTestUtil.FIELDS
+				).put(
+					"fuzziness", "AUTO"
+				).put(
+					"keywords", "${keywords}"
+				).put(
+					"minimum_should_match", 0
+				).put(
+					"operator", "or"
+				).put(
+					"slop", 0
+				).put(
+					"type", "best_fields"
+				).build()
+			},
+			new String[] {"Text Match Over Multiple Fields"});
+
+		_keywords = "Article";
+
+		_assertSearch("[Article 1.2, Article 1.1, Article 1.0]");
```

Comment:
Looks potentially flaky, don't we want to use _assertSearchIgnoreRelevance here?

Reply:
Yeah make sense